src/main/java/bio/terra/service/dataset/DatasetJsonConversion.java

```diff
@@ -16,6 +16,7 @@
 import bio.terra.model.RelationshipModel;
 import bio.terra.model.RelationshipTermModel;
 import bio.terra.model.TableModel;
+import bio.terra.model.TableDataType;
```

Comment:
I think this import is no longer needed.

Reply:
Indeed, these will cause errors with checkstyles, hence the current GitHub action failures.

```
[ERROR] /github/workspace/src/main/java/bio/terra/service/dataset/DatasetJsonConversion.java:19:8: Unused import - bio.terra.model.TableDataType. [UnusedImports]
[ERROR] /github/workspace/src/main/java/bio/terra/service/dataset/DatasetRequestValidator.java:22:8: Unused import - java.lang.reflect.Array. [UnusedImports]
```